src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/GenericWhitespaceCheck.java

```diff
@@ -71,10 +77,17 @@
  * MyClass obj = new &lt;String&gt;MyClass();
  * </pre>
  * <p>
- * To configure the check:
+ * Examples with incorrect spacing:
  * </p>
  * <pre>
- * &lt;module name=&quot;GenericWhitespace&quot;/&gt;
+ * Box&lt;String&gt; b = Box. &lt;String&gt;of("foo"); // violation, "&lt;" preceded with whitespace
```

Comment:
`Box&lt;String&gt;` violation here is in right side of expression, so to make it clearer use '<' and '>' only in place where violation occurs.
For example, here it can be like
`Box b = // here is code with violation`

Reply:
Noted, will fix